src/misc/parse_arguments.cpp

```diff
@@ -187,6 +187,12 @@ argparse::ArgumentParser parsed_arguments(
   args.plot_quadtree = arguments.get<bool>("--plot_quadtree");
   args.output_shifted_insets =
     arguments.get<bool>("--output_shifted_insets");
+
+  args.id_col = arguments.present<std::string>("--id");
+  args.area_col = arguments.present<std::string>("--area");
```

Comment:
Looks good code-wise, but a code comment may be helpful here to explain that `.present` returns an `optional`, and not a boolean as its name might suggest.